unity-client/Assets/Scripts/MainScripts/DCL/Controllers/AssetManager/AssetBundles/AB/AssetBundlesLoader.cs

```diff
@@ -87,13 +87,13 @@ private IEnumerator LoadAssetBundlesCoroutine()
                 while (assetBundlesReadyToBeLoaded.Count > 0)
                 {
                     assetBundleInfoToLoad = assetBundlesReadyToBeLoaded.Dequeue();
-                    yield return LoadAssetsInOrder(assetBundleInfoToLoad, SKIPPED_FRAMES_AFTER_BUDGET_TIME_FOR_NEARBY_ASSETS);
+                    yield return LoadAssetsInOrder(assetBundleInfoToLoad, SKIPPED_FRAMES_AFTER_BUDGET_TIME_IS_REACHED_FOR_NEARBY_ASSETS);
                 }
 
                 while (assetBundlesWaitingForLoad.Count > 0 && assetBundlesReadyToBeLoaded.Count == 0)
                 {
                     assetBundleInfoToLoad = assetBundlesWaitingForLoad.Dequeue();
-                    yield return LoadAssetsInOrder(assetBundleInfoToLoad, SKIPPED_FRAMES_AFTER_BUDGET_TIME_FOR_DISTANT_ASSETS);
+                    yield return LoadAssetsInOrder(assetBundleInfoToLoad, SKIPPED_FRAMES_AFTER_BUDGET_TIME_IS_REACHED_FOR_DISTANT_ASSETS);
```

Comment:
What happens if any asset inside this low priority queue is loading slowly and we get near to it? It shouldn't be promoted to the high priority one?

Reply:
Yeah, I already had that case in mind and I was planning, as a second iteration, to implement some kind of system able to prioritize the loading of some bundles from the "low priority" queue. But I still don't have a final solution to apply.

Reply:
Are you sure we can push this to a second iteration? If we have ABs with a big asset list, this can downgrade the experience too much. 

i.e.:
* User starts exploring the world
* AB start downloading from far away, slowly
* User gets near the scene before the ABs finish loading
* User has to wait too much time for them to finish because they were enqueued in the low priority list

Reply:
With "second iteration" I wanted to say "I'll do it during this implementation, after these first tests", sorry for the confussion :D

Reply:
I have added a system for re-prioritize the "low priority" queue. The approach is: each time a new AB is marked for load, before add it to any queue, we review the "low priority" queue and we go taking ABs while their distances to the player are less than the configured limit (`MAX_SQR_DISTANCE_FOR_QUICK_LOADING`). All these taken AB's are added in the "hig priority" queue.

In order to avoid making too many calls to this "reprioritization" function, I have added a `TIME_BETWEEN_REPRIORITIZATIONS` that will indicates how many time should pass between each re-prioritization.